locale/en.po

```diff
@@ -2167,19 +2167,23 @@ msgstr "Art Gallery"
 msgid "LICENSES"
 msgstr "Licenses"
 
-#: ../src/general/MainMenu.tscn:496
+#: ../src/general/MainMenu.tscn:420
+msgid "SUGGESTIONS"
```

Comment:
The key should be "VISIT_SUGGESTIONS_SITE" and the English text "Suggest a Feature"